pom.xml

```diff
@@ -36,7 +36,7 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Wow, really being a long time since i have seen a pom.xml, any time this should be moving to gradle or not?

Reply:
Hahaha yeah we probably should, in a different PR though